help/en/html/doc/Technical/TimeProviders.shtml

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>JMRI: Time providers</title>
```

Comment:
Thanks for writing this great page.  Can I suggestion you link it from the developer's pages sidebar by adding a link in help/en/html/doc/Technical/Sidebar.shtml?

Reply:
Thanks! Fixed.